src/views/profile/profile.ejs

```diff
@@ -1,536 +1,569 @@
 <%- include('../partials/header') %>
 
-<link rel="stylesheet" type="text/css" href="/stylesheets/profile.css">
-<link rel="stylesheet" type="text/css" href="/libraries/flag-icon/css/flag-icon.min.css">
-
-<script src="https://cdnjs.cloudflare.com/ajax/libs/1000hz-bootstrap-validator/0.11.9/validator.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.2/Chart.min.js"></script>
-
-
-<%
-const sec = 1000;
-const min = sec * 60;
-const hour = min * 60;
-const day = hour * 24;
-const month = day * 30;
-const year = month * 12;
-
-
-var getDatesString = function(time) {
-
-    var varTime = time;
-
-    var str = "";
-
-    if (Math.floor(varTime / year) !== 0) {
-        str += Math.floor(varTime / year) + " yrs, ";
-        varTime = varTime - Math.floor(varTime / year) * year;
-    }
-    if (Math.floor(varTime / month) !== 0) {
-        str += Math.floor(varTime / month) + " mths, ";
-        varTime = varTime - Math.floor(varTime / month) * month;
-    }
-    if (Math.floor(varTime / day) !== 0) {
-        str += Math.floor(varTime / day) + " days, ";
-        varTime = varTime - Math.floor(varTime / day) * day;
-    }
-    if (Math.floor(varTime / hour) !== 0) {
-        str += Math.floor(varTime / hour) + " hrs, ";
-        varTime = varTime - Math.floor(varTime / hour) * hour;
-    }
-    if (Math.floor(varTime / min) !== 0) {
-        str += Math.floor(varTime / min) + " mins, ";
-        varTime = varTime - Math.floor(varTime / min) * min;
-    }
-    if (Math.floor(varTime / sec) !== 0) {
-        str += Math.floor(varTime / sec) + " secs. ";
-        varTime = varTime - Math.floor(varTime / sec) * sec;
-    }
-
-    return str;
-}
-%>
-
-
-<div class="container myContainer">
-    <div class="row">
+  <link rel="stylesheet" type="text/css" href="/stylesheets/profile.css">
```

Comment:
Can you undo the formatting changes here please? It's hard to see what you've actually changed.

Reply:
Yep, I have already undo it. and submitted it again. I guess it's because I copy the whole file and then paste it.